db/migrate/20241017172408_remove_unique_title_index_from_evaluation_criteria.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveUniqueTitleIndexFromEvaluationCriteria < ActiveRecord::Migration[7.2]
+  def change
+    # remove_index :evaluation_criteria, name: 'index_evaluation_criteria_on_evaluation_form_id_and_title'
+    remove_index :evaluation_criteria, [:evaluation_form_id, :title]
```

Comment:
is there a reason to remove this index?

Reply:
oh I remember discussing this, about deleting existing criteria and adding a new one with the same name 👌